spec.md

```diff
@@ -1066,6 +1066,25 @@ MUST follow the same style rules as other binary [comparison][] operators:
 $variable = $foo ?: 'bar';
 ```
 
+### 6.4. Operator's placement
+
+When a statement that includes an operator must be split into multiple lines,
+the operator SHOULD be placed at the beginning of the new line.
```

Comment:
This should also mention the indentation, explicitly.

I am also tempted to make this a MUST, but would like input from the rest of the WG.

Reply:
@Crell not sure if I understand what you mean with `mention explicitly the identation`.
Do you mean something like?

```
When a statement that includes an operator must be split into multiple lines,
the operator SHOULD be placed at the beginning of the new line preceded
by 4 spaces.
```

Regarding the `SHOULD` or `MUST`, I think `MUST` could work better here 🤔

Reply:
Consider this example for lists of implements:

> Lists of implements and, in the case of interfaces, extends MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one interface per line. For example:

I think we probably want most of these things for multiline ternaries too:
- "MAY be split across multiple lines" It's important we allow this explicitly with an RFC 2119 word
- "where each subsequent line is indented once" Require the indentation
- "MUST be only one interface per line" When split on multiple lines, ternaries must occupy at least 3 lines, never 2.

So the golden path is allowed explicitly and the following ends up disallowed explicitly:
```php
$foo = $baz
? 'fizz'
: 'buzz';

$foo = $baz
    ? 'fizz' : 'buzz';

$foo = $baz
       ? 'fizz' 
       : 'buzz';

Reply:
It should be better now. If you still think something could be improved, let me know 😃